providers/implementations/rands/drbg_hmac.c

```diff
@@ -484,6 +484,13 @@ static int drbg_hmac_set_ctx_params_locked
     if (!OSSL_FIPS_IND_SET_CTX_FROM_PARAM(ctx, OSSL_FIPS_IND_SETTABLE0, p->ind_d))
         return 0;
 
+    /* Check for errant MAC parameter */
```

Comment:
We should just ignore that parameter altogether..
ossl_prov_macctx_load should pass NULL instead of p->mac and then set the macname param to "HMAC"

Reply:
For this case that should not even be a settable.

Reply:
i.e. It shoul dbe removed from inc.in

Reply:
Both setting and getting were allowed.  There isn't any harm having it gettable although it's not that useful.

I think I've come up with a situation where this isn't good.